assets/hip-1056/protobuf/services/contract_types.proto

```diff
@@ -0,0 +1,80 @@
+/**
+ * # Contract Message Types
+ * Message types used in contract transactions.
+ *
+ * ### Keywords
+ * The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT",
+ * "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this
+ * document are to be interpreted as described in
+ * [RFC2119](https://www.ietf.org/rfc/rfc2119) and clarified in
+ * [RFC8174](https://www.ietf.org/rfc/rfc8174).
+ */
+syntax = "proto3";
+
+package proto;
+
+// SPDX-License-Identifier: Apache-2.0
+option java_package = "com.hederahashgraph.api.proto.java";
+// <<<pbj.java_package = "com.hedera.hapi.node.contract">>> This comment is special code for setting PBJ Compiler java package
+option java_multiple_files = true;
+
+import "services/basic_types.proto";
+import "google/protobuf/wrappers.proto";
+
+/**
+ * Context of an internal call in an EVM transaction that is not otherwise externalized.<br/>
```

Comment:
Q: By internal do you mean Hiero internal, for instance this is a child Hiero transaction and not an internal EVM contract to contract call.
We should disambiguate between the two if possible